chpl/chpl-api/e2e/collections/developer-controller.postman_collection.json

```diff
@@ -49,13 +98,202 @@
+							"        \"id\": 0,",
+							"        \"status\": \"string\"",
+							"      },",
+							"      \"statusEvents\": [",
+							"        {",
+							"          \"developerId\": 0,",
+							"          \"id\": 0,",
+							"          \"reason\": \"string\",",
+							"          \"status\": {",
+							"            \"id\": 0,",
+							"            \"status\": \"string\"",
+							"          },",
+							"          \"statusDate\": \"2020-10-09T17:08:51.022Z\"",
+							"        }",
+							"      ],",
+							"      \"transparencyAttestations\": [",
+							"        {",
+							"          \"acbId\": 0,",
+							"          \"acbName\": \"string\",",
+							"          \"attestation\": {",
+							"            \"removed\": false,",
+							"            \"transparencyAttestation\": \"string\"",
+							"          }",
+							"        }",
+							"      ],",
+							"      \"website\": \"string\"",
+							"    }",
+							"  ]",
+							"}",
+							"var jsonResponse = pm.response.json();",
+							"pm.test('developer schema is valid', function() {",
+							"  pm.expect(tv4.validate(jsonResponse, developerSchema)).to.be.true;  ",
+							"});",
+							"",
+							"pm.test(\"developer contact details should not include friendly name\", function () {",
+							"    pm.expect(pm.response.text()).to.not.include(\"friendlyName\");",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "API-KEY",
+						"value": "{{apiKey}}",
+						"type": "text"
+					},
+					{
+						"key": "Content-Type",
+						"value": "application/json",
+						"type": "text"
+					},
+					{
+						"key": "Authorization",
+						"value": "{{authToken}}",
+						"type": "text",
+						"disabled": true
+					}
+				],
+				"url": {
+					"raw": "{{url}}/developers/{{developerId}}",
+					"host": [
+						"{{url}}"
+					],
+					"path": [
+					    "rest",
+						"developers",
+						"{{developerId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "developer edit response validation_PUT_/developers",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "0ea2a849-5942-44e3-8d2f-877077e43bf8",
+						"exec": [
+							"\r",
+							"pm.test(\"valid developer edit request should return valid response\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"\r",
+							"pm.test(\"validate edit response-developer name is correct\", function () {\r",
+							"    var jsonData = pm.response.json();\r",
+							"    pm.expect(jsonData.name).to.eql('DSS, Inc');\r",
+							"});\r",
+							"\r",
+							"pm.test(\"developer contact details should not include friendly name\", function () {\r",
+							"    pm.expect(pm.response.text()).to.not.include(\"friendlyName\");\r",
+							"});\r",
+							""
+						],
```

Comment:
Need a test to check status code. As {{developerId}} is not being passed, request status code is coming as 400.

Reply:
developerId is hard coded now.